bokehjs/src/lib/models/glyphs/marker.ts

```diff
@@ -127,6 +128,17 @@ export abstract class MarkerView extends XYGlyphView {
     return new Selection({indices})
   }
 
+  override _set_data(indices: number[] | null): void {
+    super._set_data(indices)
+
+    const max_size = (() => {
+      const {size} = this
+      return size.is_Scalar() ? size.value : max((size as p.UniformVector<number>).array)
+    })()
```

Comment:
With PR #12677 merged, this block can now be replaced with `const max_size = uniforms.max(this.size)` (where `uniforms` comes from `import * as uniforms from "core/uniforms"`).